infra/modules/aws-psoxy-bulk-existing/README.md

```diff
@@ -0,0 +1,48 @@
+# aws-hris
+
+This example provisions an AWS Lambda that pseudonomizes files uploaded into the bucket. The source file is dropped in a *input bucket* and
+result will appear in *output bucket*.
+
+## Authentication
+
+### AWS
+Follow [`docs/aws/getting-started.md`](../../../docs/aws/getting-started.md) to setup AWS CLI to
+authenticate as a user/role which can access/create the AWS account in which you wish to provision
+your psoxy instance.
+
+## Configuration
+
+Example `terraform.tfvars`:
+```terraform
+aws_account_id                = "123456789"
+aws_assume_role_arn           = "arn:aws:iam::123456789:role/InfraAdmin"
+environment_name              = "dev-aws"
+bucket_prefix                 = "some_prefix_for_bucket"
+caller_aws_account_id         = "914358739851:root"
+caller_external_user_id       = "your-worklytics-service-account-id"
+source_kind                   = "hris"
```

Comment:
don't you need in this case a s3 bucket input as mandatory??
`input_bucket = "your S3 bucket name" ?`